tests/testthat/helper-aaa.R

```diff
@@ -1,10 +1,8 @@
 
 teardown_env <- function() {
-  if (interactive()) globalenv() else testthat::teardown_env()
-}
-
-`_renv_tests_repopath` <- renv_scope_tempfile("renv-repos-", envir = teardown_env())
-
-renv_tests_repopath <- function() {
-  `_renv_tests_repopath`
+  if (is.null(renv_namespace_load("testthat")$testthat_env$teardown_env)) {
```

Comment:
I'll come back to this at some point, but this quick hack makes the test file button work for me again.